cli/cli_util.c

```diff
@@ -139,8 +144,13 @@ kvdb_compact_request(const char *kvdb_home, const char *request_type, u32 timeou
 
         for (size_t i = 0; i < namec; i++) {
             struct hse_kvs *k;
+            const char *kvs_paramv[] = { "cn_close_wait=true" };
```

Comment:
Should close-wait be contigent on "request_full" ?

Reply:
Yes. I think I addressed this in another comment.